libs/providers/flagd/src/proto/ts/schema/v1/schema.client.ts

```diff
@@ -0,0 +1,76 @@
+// @generated by protobuf-ts 2.2.2 with parameter long_type_string,generate_dependencies
```

Comment:
Avoid version controlling generated code.

Reply:
removed + updated nx to include generation step on test lint and package actions